### ISSUE TITLE
I know I am stupid but...

### ISSUE THREAD

**Issue by Luke2Sky** [RyotaUshio/obsidian-math-booster#172]
When I activated the plugin I lost a lot of the file I was working on. I don't get why. I should've made a backup since this is a beta plugin...

**Comment by Luke2Sky** [comment]
Update, I deactivated it and the data just came back. Will close and leave it here in the hopes of helping someone.

**Comment by RyotaUshio** [comment]
Hi, sorry for the data loss. I'm relieved to hear your data came back.

Honestly, I have no idea why this happened. It has never happened to me, and also this is the first time I received a report like this.

Could you provide a little more information? For example:
- Obsidian version
- Plugin version
- Platform (OS)
- The situation where the data loss occurred (did it happen the moment you enabled the plugin?)

**Comment by RyotaUshio** [comment]
My current guess is that you didn't actually lose the data, but the plugin's CSS code made it invisible.
This can happen when, for example, your note contains invalid math delimiters like the following:

1. Unmatched delimiter ($)
```
$$$
```

2. Empty inline math inside display math
```
$$
\text{$$}
$$
```

This issue is planned to be solved in the next release.

**Comment by Luke2Sky** [comment]
I think the fault was something I described with more detail in another issue. It is the way I format my notes. 
Version: 1.4.5
plugin version: I just downloaded it using BRAT
Plataform: Manjaro - Gnome
Yes, it happaned the moment I enabled it
Only the active note, thank goodness... 
This is my backup after what happened:
![image](https://github.com/RyotaUshio/obsidian-math-booster/assets/61480539/0a09d8e1-9aac-4b31-b5a1-10737d6ade9f) 
This is what happened when I activated the plugin. I think I found the root cause of the problem. If you see the other issue, one thing in particular seems to have caused it. 
![image](https://github.com/RyotaUshio/obsidian-math-booster/assets/61480539/5fd33291-cf77-4327-b8a3-88540b049984)
